### ISSUE TITLE
Getting 0x13a error when uploading

### ISSUE THREAD

**Issue by lukakrz** [metaplex-foundation/metaplex#1405]
Hi i getting 0x13a error when i trying to uploading. Could you help me solve this problem? 

```
Beginning the upload for 2 (png+json) pairs
started at: 1641398101839
wallet public key: Fnw6jMzwpJ8BPqDb9tditNXKV5CLwYeMCW8A856zgTfv
Processing file: 0
initializing config
Transaction simulation failed: Error processing Instruction 1: custom program error: 0x13a 
    Program 11111111111111111111111111111111 invoke [1]
    Program 11111111111111111111111111111111 success
    Program cndyAnrLdpjq1Ssp1z8xxDsB8dxe7u4HL5Nxi2K5WXZ invoke [1]
    Program log: Custom program error: 0x13a
    Program cndyAnrLdpjq1Ssp1z8xxDsB8dxe7u4HL5Nxi2K5WXZ consumed 6918 of 200000 compute units
    Program cndyAnrLdpjq1Ssp1z8xxDsB8dxe7u4HL5Nxi2K5WXZ failed: custom program error: 0x13a
Translating error SendTransactionError: failed to send transaction: Transaction simulation failed: Error processing Instruction 1: custom program error: 0x13a
    at Connection.sendEncodedTransaction (C:\Users\Lukasz\Desktop\metaplex-1.0.0\js\node_modules\@solana\web3.js\src\connection.ts:3553:13)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async Connection.sendRawTransaction (C:\Users\Lukasz\Desktop\metaplex-1.0.0\js\node_modules\@solana\web3.js\src\connection.ts:3513:20)
    at async sendAndConfirmRawTransaction (C:\Users\Lukasz\Desktop\metaplex-1.0.0\js\node_modules\@solana\web3.js\src\util\send-and-confirm-raw-transaction.ts:27:21)
    at async Provider.send (C:\Users\Lukasz\Desktop\metaplex-1.0.0\js\packages\cli\node_modules\@project-serum\anchor\src\provider.ts:114:18)
    at async Object.rpc [as initializeConfig] (C:\Users\Lukasz\Desktop\metaplex-1.0.0\js\packages\cli\node_modules\@project-serum\anchor\src\program\namespace\rpc.ts:19:23) {
  logs: [
    'Program 11111111111111111111111111111111 invoke [1]',
    'Program 11111111111111111111111111111111 success',
    'Program cndyAnrLdpjq1Ssp1z8xxDsB8dxe7u4HL5Nxi2K5WXZ invoke [1]',
    'Program log: Custom program error: 0x13a',
    'Program cndyAnrLdpjq1Ssp1z8xxDsB8dxe7u4HL5Nxi2K5WXZ consumed 6918 of 200000 compute units',
    'Program cndyAnrLdpjq1Ssp1z8xxDsB8dxe7u4HL5Nxi2K5WXZ failed: custom program error: 0x13a'
  ]
}
Error deploying config to Solana network. SendTransactionError: failed to send transaction: Transaction simulation failed: Error processing Instruction 1: custom program error: 0x13a
    'Program cndyAnrLdpjq1Ssp1z8xxDsB8dxe7u4HL5Nxi2K5WXZ invoke [1]',
    'Program log: Custom program error: 0x13a',
    'Program cndyAnrLdpjq1Ssp1z8xxDsB8dxe7u4HL5Nxi2K5WXZ consumed 6918 of 200000 compute units',
    'Program cndyAnrLdpjq1Ssp1z8xxDsB8dxe7u4HL5Nxi2K5WXZ failed: custom program error: 0x13a'
  ]
}
C:\Users\Lukasz\Desktop\metaplex-1.0.0\js\node_modules\@solana\web3.js\src\connection.ts:3553
      throw new SendTransactionError(
            ^
SendTransactionError: failed to send transaction: Transaction simulation failed: Error processing Instruction 1: custom program error: 0x13a
    at Connection.sendEncodedTransaction (C:\Users\Lukasz\Desktop\metaplex-1.0.0\js\node_modules\@solana\web3.js\src\connection.ts:3553:13)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async Connection.sendRawTransaction (C:\Users\Lukasz\Desktop\metaplex-1.0.0\js\node_modules\@solana\web3.js\src\connection.ts:3513:20)
    at async sendAndConfirmRawTransaction (C:\Users\Lukasz\Desktop\metaplex-1.0.0\js\node_modules\@solana\web3.js\src\util\send-and-confirm-raw-transaction.ts:27:21)
    at async Provider.send (C:\Users\Lukasz\Desktop\metaplex-1.0.0\js\packages\cli\node_modules\@project-serum\anchor\src\provider.ts:114:18)
    at async Object.rpc [as initializeConfig] (C:\Users\Lukasz\Desktop\metaplex-1.0.0\js\packages\cli\node_modules\@project-serum\anchor\src\program\namespace\rpc.ts:19:23) {
  logs: [
    'Program 11111111111111111111111111111111 invoke [1]',
    'Program 11111111111111111111111111111111 success',
    'Program cndyAnrLdpjq1Ssp1z8xxDsB8dxe7u4HL5Nxi2K5WXZ invoke [1]',
    'Program log: Custom program error: 0x13a',
    'Program cndyAnrLdpjq1Ssp1z8xxDsB8dxe7u4HL5Nxi2K5WXZ consumed 6918 of 200000 compute units',
    'Program cndyAnrLdpjq1Ssp1z8xxDsB8dxe7u4HL5Nxi2K5WXZ failed: custom program error: 0x13a'
  ]
}
```

**Comment by jiaxiangcheng** [comment]
same error when i try to upload to mainnet

**Comment by prashantcrypto** [comment]
I can make it.
I takes just 1 sol for all.
Upto creating minting site ready by cmv2

My discord id shyaid#3516

**Comment by prashantcrypto** [comment]
Message me on my discord i will tell everything there about this

**Comment by prashantcrypto** [comment]
Just add cm v2 config file ther

**Comment by Raziel-Zavaleta** [comment]
v1 is not longer abilable, use v2

**Comment by utgarda** [comment]
Is there a possibility to learn from the code of from help messages that `0x13a` means "v1 is deprecated", or is this issue together with Discord the source of truth?

**Comment by raryson** [comment]
How i can fix? I have try search for shyaid#3516 on disorcd to ask or how use v2 but i dont found nothing

**Comment by prashantcrypto** [comment]
> How i can fix? I have try search for shyaid#3516 on disorcd to ask or how use v2 but i dont found nothing

Sorry discord is shyaid#5618.

Sorry for writing mistake

**Comment by prashantcrypto** [comment]
> How i can fix? I have try search for shyaid#3516 on disorcd to ask or how use v2 but i dont found nothing

Sorry


Please message to shyaid#5168
